core/trino-main/src/main/java/io/trino/connector/InternalMetadataProvider.java

```diff
@@ -37,10 +37,10 @@
 public class InternalMetadataProvider
         implements MetadataProvider
 {
-    private final MetadataManager metadataManager;
+    private final Metadata metadataManager;
```

Comment:
Is the name `metadataManager` left on purpose like this during the switch to the `Metadata` interface?

Reply:
same for constructor argument name